src/azul/drs.py

```diff
@@ -185,6 +177,42 @@ def drs_object(object_uuid, version=None):
     }
 
 
+class AccessMethod(Enum):
```

Comment:
For tying replica and scheme and avoiding having to duplicate the name for the enum value, this might be helpful:

```python
from collections import namedtuple
from enum import Enum


class AccessMethod(namedtuple('AccessMethod', 'scheme replica'), Enum):
    gs = ('gs', 'gcp')
    https = ('https', 'aws')

    def __str__(self) -> str:
        return self.name

print(AccessMethod.gs)
print(repr(AccessMethod.gs))
print(AccessMethod.gs.replica)
```

produces

```
gs
<AccessMethod.gs: AccessMethod(scheme='gs', replica='gcp')>
gcp
```

Reply:
Thanks for this! I had written something similar before, but I was unsatisfied with having to call `AccessMethod.gs.name` for example. Defining `__str__` is simple and clear.

Also I didn't realize you could inherit from both namedtuple, and Enum. It makes this implementation much simpler!